sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # Release History
 
-## 2.2.1 (Unreleased)
+## 3.0.0-beta.1 (Unreleased)
 
 ### Features Added
 
 ### Breaking Changes
 
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
```

Comment:
should be it under the added features?

Reply:
@AikoBB ARB asked to highlight that this is a breaking change

Reply:
It is a feature that introduces a breaking change.
I would add it as a feature (because people will look for it within Features). And additionally, add a record under breaking changes explaining when the addition of a new identifier can be breaking in some cases - e.g. if someone was consuming unknown identifiers with rawId starting with "28:" now these will become a standalone type...so if they had a type-based switch, they'd need to adjust it, etc.

Reply:
@petrsvihlik updated